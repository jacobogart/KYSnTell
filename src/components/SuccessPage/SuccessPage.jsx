import React from 'react';

export const SuccessPage = (props) => {
  return (
    <div className='SuccessPage'>
      <h3>Messages successfully sent!</h3>
      <p>Thank you for taking the brave step to share your status with your partner(s).</p>
    </div>
  )
}
