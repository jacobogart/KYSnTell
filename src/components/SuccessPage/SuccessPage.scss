.SuccessPage {
  text-align: center;
  padding: 35px;

  h3 {
    font-size: 50px;
    color: rgb(91, 206, 91)
  }
}