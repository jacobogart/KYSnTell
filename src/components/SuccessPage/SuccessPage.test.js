import React from 'react';
import { shallow } from 'enzyme';
import { SuccessPage } from './SuccessPage';

describe('SuccessPage', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<SuccessPage />)
    expect(wrapper).toMatchSnapshot();
  });
});