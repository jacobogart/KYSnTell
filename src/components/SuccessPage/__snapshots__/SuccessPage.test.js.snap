// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessPage should match snapshot 1`] = `
<div
  className="SuccessPage"
>
  <h3>
    Messages successfully sent!
  </h3>
  <p>
    Thank you for taking the brave step to share your status with your partner(s).
  </p>
</div>
`;
